bundles/specmate-migration/src/com/specmate/migration/internal/services/MigratorService.java

```diff
@@ -92,6 +92,7 @@ public boolean needsMigration() throws SpecmateException {
 			// TODO using the exception for logic decisions is not good as we

```

Comment:
should we act on this todo now? Otherwise it will probably never happen.

Reply:
Should be fixed in PR 140.